bigquery/storage/managedwriter/connection.go

```diff
@@ -494,18 +494,29 @@ func connRecvProcessor(ctx context.Context, co *connection, arc storagepb.BigQue
 			resp, err := arc.Recv()
 			co.release(nextWrite)
 			if err != nil {
+				// The Recv() itself yielded an error.  We increment AppendResponseErrors by one, tagged by the status
+				// code.
+				status := grpcstatus.Convert(err)
```

Comment:
I wonder if `status` can be `nil` here as this `err` is coming from a grpc stream method. Might worth add the extra check

Reply:
an `io.EOF` will probably cause `grpcstatus.Convert` to return `nil`

Reply:
It wraps the error as a code.Unknown if it doesn't embed an rpc status error, ex: https://github.com/grpc/grpc-go/blob/faab8736bf73291f92b867d5dae31c927d53d508/status/status.go#L96-L125